src/ert/validation/range_string_argument.py

```diff
@@ -49,15 +50,31 @@ def validate(self, token: str) -> ValidationStatus:
 
 
 class NotInStorage(ArgumentDefinition):
+    NOT_A_VALID_NAME_FORMAT = (
```

Comment:
Does it work to subclass `ProperNameFormatArgument` instead so we dont need to duplicate this? Or just import it from there: ProperNameFormatArgument.PATTERN